WhenPushComesToShove/Assets/3-Scripts/Player Scripts/PlayerMovementScript.cs

```diff
@@ -83,4 +88,24 @@ public Vector2 GetAimDirection()
     {
         return aimInputVector;
     }
+
+    //Helper Functions for Modifying Move Speed
+    public void ChangeMoveSpeed(float newSpeed)
+    {
+        moveSpeed = newSpeed;

```

Comment:
This stuff will have to change with the new player movement. Shouldn't be too hard, but I will hold off merging until its set up with the physics based controller